.github/.release.yml

```diff
@@ -0,0 +1,20 @@
+changelog:
+  exclude:
+    labels:
+      - Skip-Release-Notes
+  categories:
+    - title: Bugfixes
+      labels:
+        - Bug-Fix
+    - title: New Features
+      labels:
+        - New Feature
+    - title: Enhancements
+      labels:
+        - Enhancement
+    - title: Not Yet Enabled
+      labels:
+        - Not-Yet-Enabled
+    - title: Other
+      labels:
+        - "*"
```

Comment:
Wouldn't this put all labels under Other?

Reply:
no, this is a catch all. It only does this if it hasn't matched anything else above